### ISSUE TITLE
[Spree Upgrade] Broken Enterprise Permissions tab (1 broken test)

### ISSUE THREAD

**Issue by luisramos0** [openfoodfoundation/openfoodnetwork#3073]
I am not sure what's going on with Spree translations in both v1 and v2 (this is breaking a test in spree upgrade build). We need to fix quite a few broken translations in the BackOffice.

## What
First v1 (ofn master branch),
Katuma Staging:
<img width="334" alt="screenshot 2018-11-17 at 14 29 01" src="https://user-images.githubusercontent.com/1640378/48662137-792f0d00-ea75-11e8-9916-49a7abdcfc8a.png">

Katuma staging:
```
openfoodnetwork@staging:~/apps/openfoodnetwork/current$ cat config/locales/es.yml | grep enterprise_relationships
  enterprise_relationships: "Permisos de la organización"
```

And then v2 (ofn spree upgrade branch),
This is the spec broken in the spree upgrade build: sp spec/features/admin/enterprise_relationships_spec.rb.

## Possible solution
I think there is a problem with the solution of  #1384

In app/views/admin/shared/_enterprises_sub_menu.html.haml
We have
`= tab :enterprise_relationships, url: main_app.admin_enterprise_relationships_path`

And this constant :enterprise_relationships is not being translated. I tried adding a t() around it but that breaks the code. I tried moving the translations from root to admin: shared: in en.yml but that doesn't work.

I am not yet sure how to fix this. @kristinalim @mkllnk any ideas?

**Comment by luisramos0** [comment]
This is a broken test in the spree upgrade build but also a translations bug in Katuma (it's also happening in live). So bug-s4.

**Comment by luisramos0** [comment]
ok, in Spree step-6a:
https://github.com/openfoodfoundation/spree/blob/step-6a/backend/app/helpers/spree/admin/navigation_helper.rb#L18
navigation_helper is using t()
but in 2-0-4:
https://github.com/openfoodfoundation/spree/blob/2-0-4-stable/backend/app/helpers/spree/admin/navigation_helper.rb#L21
navigation_helper is using Spree.t() and Spree.t() doesnt catch any ofn translations like "enterprise_relationships"

This explains why it's totally broken in the spree upgrade build, the base en.yml translation from "enterprise relationship" to "enterprise permissions" is not seen and the test fails.

I am still trying to understand why the step-6a version is also failing to translate. It's spree code but it's calling t() so the ofn translation to spanish should be picked up.

**Comment by mkllnk** [comment]
I think I know what the problem in Spree2 is: `:scope => [:admin, :tab]`. The translations have to be in a different scope, they are not global any more. They have to be in `admin.tab`, not `admin.shared`.

I don't know why you are experiencing problems with Spree1 as well. Caching issue?

**Comment by sauloperez** [comment]
> Caching issue? 

Could be. I see it working on production

![screenshot from 2018-11-19 13-58-55](https://user-images.githubusercontent.com/762088/48708605-4adf3800-ec03-11e8-8206-37f782e8f04b.png)

**Comment by luisramos0** [comment]
what the hell is going on here? @sauloperez can you share the URL where you saw that?
am I looking at a diff katuma live?
<img width="760" alt="screenshot 2018-11-19 at 13 02 24" src="https://user-images.githubusercontent.com/1640378/48708808-79f1ab80-ebfb-11e8-9bc5-01f5bb79d7ab.png">

I cleared browser cache data and I am still seeing this.

**Comment by luisramos0** [comment]
ah, ok, I am looking at Portuguese katuma... my mistake.

**Comment by luisramos0** [comment]
So, this issue is only in Spree 2 as the v1 issue was caused just because I had Portuguese selected in katuma.

I'll try what @mkllnk suggested (scope) and report back.

thanks for the help guys!

**Comment by luisramos0** [comment]
this will be fixed as part of #3119
